firecracker-control/local.go

```diff
@@ -206,11 +214,22 @@ func (s *local) startMachine(ctx context.Context, id string, machine machineIfac
 		return err
 	}
 
+	if err := s.publisher.Publish(ctx, startEventName, &proto.VMStart{VMID: id}); err != nil {
+		logger.WithError(err).Error("failed to publish create event")
+		return err
```

Comment:
Should we actually fail the startMachine operation if we fail to publish the event?

If the event publish is just for informational purposes, it seems like it's enough to log the error but move on. Do we rely on the event publish for basic functionality anywhere?

Reply:
I don't know the exact use key for events yet. I'll clarify and will update this behavior. Let's leave it as is for now.